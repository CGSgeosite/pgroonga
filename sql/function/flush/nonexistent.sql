SELECT pgroonga.flush('pgroonga_index');
