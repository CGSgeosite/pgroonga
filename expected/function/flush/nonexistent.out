SELECT pgroonga.flush('pgroonga_index');
ERROR:  relation "pgroonga_index" does not exist
